http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -461,6 +461,15 @@ paths:
           schema:
             type: boolean
             default: false
```

Comment:
think the `listRuntime` and `listDisk` responses need to be udpated as well

Reply:
OH i see what you mean - i still need to update the responses. will do!